.ci/deps.minimal.sh

```diff
@@ -0,0 +1 @@
+deps.generic.sh
```

Comment:
use `source` instead of unix only symlinks?

Reply:
For minimal and generic, which are Travis specific names, symlinks seem ok, as there is no chance they could be used on a Windows box.